src/jit/jitconfigvalues.h

```diff
@@ -261,6 +261,8 @@ CONFIG_STRING(JitInlineReplayFile, W("JitInlineReplayFile"))
 CONFIG_INTEGER(JitInlinePolicyLegacy, W("JitInlinePolicyLegacy"), 0)
 CONFIG_INTEGER(JitInlinePolicyModel, W("JitInlinePolicyModel"), 0)
 
+CONFIG_INTEGER(JitCLRcallTimingInfo, W("JitCLRcallTimingInfo"), 0)
+
```

Comment:
The name of this strikes me as a bit weird, since in my mind, "CLR" means EE/VM + JIT + mscorlib + other base level stuff. How about just "JitCallTimingInfo"? Seems unique enough. Or, to be more explicit, JitItfCallTimingInfo or JitEECallTimingInfo.